js/components/uswds_radio_widget.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+const USWDSRadioWidget = function(props) {
+  console.log('props for radio: ', props);
+  const { enumOptions, inline } = props.options;
+
+return (
```

Comment:
Hmm, linter didn't pickup this indent error, I'll fix the linter but would you fix the indent here?